22000103/mirror.py

```diff
@@ -0,0 +1,19 @@
+T = int(input())
+
+for test_case in range(1,T+1):
+    s = str(input())
```

Comment:
깔끔한 풀이네요! 👍 
```suggestion
    s = input()
```
`input` 함수의 반환 값은 이미 string이므로, `str` 함수를 사용하지 않으셔도 괜찮습니다! 😄